backend/test/handlers/ArchiveHelper.test.ts

```diff
@@ -0,0 +1,25 @@
+// @ts-nocheck File not migrated fully to TS
+import { saveDataFromUrl } from 'handler/ArchiveHelper';
+import { request } from 'handler/RequestHandler';
+
+jest.mock('handler/ManagerHandler');
+
+jest.mock('handler/RequestHandler', () => ({
+    request: jest.fn((method, url, headers, data, onSuccess, onError) => onError())
+}));
+
+describe('ArchiveHelper', () => {
+    it('fetches extenral data with correct headers', () => {
```

Comment:
Let's fix typo at once:
```suggestion
    it('fetches external data with correct headers', () => {
```